casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -803,8 +803,7 @@ object BlockAPI {
 
   def getDataAtPar[F[_]: Concurrent: EngineCell: Log: SafetyOracle: BlockStore](
       par: Par,
-      blockHash: String,
-      usePreStateHash: Boolean
+      blockHash: String
```

Comment:
Parameter `usePreStateHash` should not be removed. It gives ability to query data on pre-state of the block.